website/content/en/docs/concepts/peers.md

```diff
@@ -109,50 +109,48 @@ multiline output of `kubectl get secret --namespace clusterlink-operator` comman
 
 {{% /expand %}}
 
-### Deploy ClusterLink via the Operator and ClusterLink CRD
+### Deploy ClusterLink via the operator and ClusterLink CR
 
 After the operator is installed, you can deploy ClusterLink by applying
- the ClusterLink instance CRD. This will cause the ClusterLink operator to
+ the ClusterLink CR. This will cause the ClusterLink operator to
  attempt reconciliation of the actual and intended ClusterLink deployment.
  By default, the operator will install the ClusterLink control and data plane
  components into a dedicated and privileged namespace (defaults to `clusterlink-system`).
- Configurations affecting the entire peer, such as the list of known Peers, are also maintained
+ Configurations affecting the entire peer, such as the list of known peers, are also maintained
  in the same namespace.
 
 Refer to the [getting started guide]({{< ref "users#setup" >}}) for a description
- of the ClusterLink instance CRD's fields.
+ of the ClusterLink CR fields.
 
 <!-- TODO expand the sample CRD file? -->
 
-## Add or remove Peers
+## Add or remove peers
 
 {{< notice info >}}
-This operation is typically done by a local **Peer administrator**, usually different
- than the **Fabric administrator**.
+This operation is typically done by a local **peer administrator**, usually different
+ than the **fabric administrator**.
 {{< /notice >}}
 
-Managing peers is done by creating, deleting and updating Peer CRD instances
+Managing peers is done by creating, deleting and updating peer CRs
  in the dedicated ClusterLink namespace (typically, `clusterlink-system`). Peers are
  added to the ClusterLink namespace by the peer administrator. Information
  regarding peer gateways and attributes is communicated out of band (e.g., provided
- by the Fabric or remote Peer administrator over email). In the future, these may
+ by the fabric or remote peer administrator over email). In the future, these may
  be configured via a management plane.
 
-There are two fundamental attributes in the Peer CRD: the Peer's name and the list of
- ClusterLink gateway endpoints through which the remote peer's Services are available.
- Peer names are unique and must align with the Subject name present in their certificate
+There are two fundamental attributes in the peer CRD: the peer name and the list of
+ ClusterLink gateway endpoints through which the remote peer services are available.
+ Peer names are unique and must align with the subject name present in their certificate
```

Comment:
nit: 
- I think we need `peer's services` as it's possessive
- the respective certificate field is called `Subject`, hence the capitalization